server/structs.py

```diff
@@ -1,10 +1,45 @@
-from typing import overload, NamedTuple
-from abc import ABCMeta
+"""
+Structs
+"""
+
+from typing import overload
+
 from pyraknet.bitstream import Serializable, c_uint8, c_uint16, c_uint32
 from enums import PACKET_IDS, PACKET_NAMES
 
+
+class WString(Serializable):
```

Comment:
This is unnecessary. `pyraknet.BitStream` reads/writes wstrings when you use a `str` object. The only reason I created CString was because it provided a more convenient and consistent wrapper around `bytes` (since that way, whatever the data was, we could just always deal with `str`s)

Reply:
Ah, didn't know that